gitgud/skills/newbasics/_firstcommit/solution.txt

```diff
@@ -0,0 +1 @@
+git gud commit
```

Comment:
When someone is solving this level, they will not run this command, they'll run something along the lines of
```
touch file
git add file
git commit -m "Initial Commit: Added file"
```